### PR TITLE
Task 239-242: TweetQA

### PR DESCRIPTION
**Pull request by Mihir3009**
1 Answer Generation Task
1 Question Generation Task
2 Classification Tasks

Please review.

### CODE REVIEW

**Review by slowy07 — COMMENTED** [review]
thanks for the contribution ! @Mihir3009 ,can you change this ??

File: tasks/README.md [hunk 1]
```diff
@@ -178,5 +178,12 @@ Name | Summary | Category
...
+`task211_logic2text_classification.json` | Given a command and corresponding interpretation, classify whether it's right interpretation or not | Classification
 `task195_sentiment140_classification.json` | Given a tweet text, classify it into positive or negative | Classification
 `task196_sentiment140_answer_generation.json` | Given a tweet text and boolean question, generate answer yes or no | Answer Generation
+`task212_logic2text_classification.json` | Given a command (in the form of logical operators), classify command in one of seven logic types | Classification
+`task239_TweetQA_answer_generation.json` | Given a context paragraph of the tweet and question, generate a right answer | Answer Generation
+`task240_TweetQA_question_generation.json` | Given a context paragraph of the tweet and answer, generate a right question | Question Generation
+`task241_TweetQA_classification.json` | Given a context paragraph of the tweet, question and corresponsing answer, generate a label whether the answer is right or wrong | Classification
```
Comment (slowy07) [inline tasks/README.md]:
```suggestion
`task241_TweetQA_classification.json` | Given a context paragraph of the tweet, question and corresponding answer, generate a label whether the answer is right or wrong | Classification
```

File: tasks/README.md [hunk 2]
```diff
@@ -178,5 +178,12 @@ Name | Summary | Category
...
+`task242_TweetQA_classification.json` | Given a context paragraph of the tweet, question and corresponsing answer, generate a label whether the context is helpful in answering question or not | Classification
```
Comment (slowy07) [inline tasks/README.md]:
```suggestion
`task242_TweetQA_classification.json` | Given a context paragraph of the tweet, question and corresponding answer, generate a label whether the context is helpful in answering question or not | Classification
```

**Review by slowy07 — COMMENTED** [review]
can you fix it ??

**Comment by danyaljj** [comment]
The PR contains some other tasks as well. Intentional?

File: tasks/README.md (hunk 2 above)
Comment (Mihir3009) [reply tasks/README.md]: Resolved

**Review by slowy07 — APPROVED** [review]
LGTM

**Comment by Mihir3009** [comment]
> The PR contains some other tasks as well. Intentional?

My Bad. Removed other tasks. Sorry for that.

**Comment by danyaljj** [comment]
@kurbster @Mirzyaaliii could you help with this PR's review?

**Comment by Mirzyaaliii** [comment]
Sort your readme file according to task numbers.

Other than this, all tasks looks good to me.

**Comment by Mihir3009** [comment]
> Sort your readme file according to task numbers.
> 
> Other than this, all tasks looks good to me.

I updated the readme file. I added my tasks accordingly. Thanks.

**Review by swarooprm — APPROVED** [review]
Looks good! Merging.
Some comments:
1. Task 241 and 242 has exactly same name, I would add few more information to make it look different.
2. In classification tasks, make sure that yes and no labels are balanced. They should not have a skewed distribution

**Pull request merged**